integration_tests/seeds/materialization/test_fct_chained_views_dependencies.csv

```diff
@@ -0,0 +1,14 @@
+parent_id,parent,child_id,child,distance
+model.dbt_project_evaluator_integration_tests.fct_model_9,fct_model_9,model.dbt_project_evaluator_integration_tests.stg_model_5,stg_model_5,1
```

Comment:
Can we remove the `parent_id` and `child_id` and just use parent and child for readability in the seed file?

Reply:
+1 !

Reply:
is the path worth adding here as well? so the user would know the exact path in case there are multiple paths between the models returned?

Reply:
Maybe worth adding to the model, but not necessary to add to the seed!